flink-core/src/main/java/org/apache/flink/configuration/AlgorithmOptions.java

```diff
@@ -60,4 +61,56 @@ public class AlgorithmOptions {
                             "Whether to use the LargeRecordHandler when spilling. If a record will not fit into the sorting"
                                     + " buffer. The record will be spilled on disk and the sorting will continue with only the key."
                                     + " The record itself will be read afterwards when merging.");
+
+    public static final ConfigOption<MemorySize> GLOBAL_AGG_BUFFER_SIZE =
```

Comment:
@twalthr I remember we had some discussions around the right place for these options.
Do you have any alternatives in mind?
How about org.apache.flink.configuration.ExecutionOptions?

Reply:
Moved to ExecutionOptions